hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/CreateTopLevelDagVisitor.java

```diff
@@ -708,7 +708,7 @@ private void connectStreamToStreamJoinInput(
             left = left.distributed().broadcast();
             right = right.unicast().local();
         } else {
-            // this strategy applies to left and inner joins non-equi joins
+            // this strategy applies to non-equi left and inner joins
```

Comment:
It would be more readable if branches on lines 707-719 were mutually exclusive. Currently we set some routing policy here, and overwrite it below.